app/jobs/deposit_status_job.rb

```diff
@@ -14,6 +14,7 @@ def perform(work:, job_id:)
     if result.success?
       work.druid = result.value!
       work.deposit!
+      StorageMigrator.migrate(druid: work.druid)
```

Comment:
Once deposited, migrate uploaded files from draft subdir to deposited subdir. See PR description for more about this.

Reply:
Won't depositing the files via SDR-API take care of this?

Reply:
Not on the H2 filesystem, I don't think so.